### ISSUE TITLE
make cuda optional

### ISSUE THREAD

**Issue by timomernick** [timomernick/pytorch-capsule#1]
i'm calling .cuda() in a few spots assuming cuda is available.

**Comment by XuecaiHu** [comment]
there is an error in your codes(In capsule_layer.py ,  c_ij = F.softmax(b_ij) ).   In the paper, $/sum_{j}c_ij = 1$
